syscoin/syscoin-api
# add project setup documentation in readme.md file

**Ready for review**

**Covers**
- README.md

**Worked**
 Write project setup documentation . api should get data from syscoin-core it has all steps of setting up syscoin-api with syscoin-core .

**Testing**

- please test all run instructions and run in different environments
- make sure all path and branch name is correct
- make sure it should not have any spell mistake
- make it should have all dependencies of project setup
- at least run twice all the setup
